py/tests/integration/common.py

```diff
@@ -130,8 +131,31 @@ def base58_decode(encoded):
     return base58.b58decode_check(encoded[3:])
 
 def encode_signed_tx(encoded_tx, signatures):
-    str = base58.b58encode_check(msgpack.packb(["sig_tx", 1, encoded_tx, signatures], use_bin_type=True))
-    return "tx$" + str
+    tag = bytearray([11])
+    vsn = bytearray([1])
+    payload = rlp.encode([tag, vsn, signatures, encoded_tx])
+    return "tx$" + base58.b58encode_check(payload)
+
+def decode_unsigned_tx(encoded_tx):
+    decoded = rlp.decode(encoded_tx)
+    tag, vsn, fields = decoded[0], decoded[1], decoded[2:]
+    # This is minimally what we need for now
+    if (tag == bytes(bytearray([42])) and vsn == bytes(bytearray([1]))):
```

Comment:
I see:
https://github.com/aeternity/epoch/blob/17a22c1e02dfae53f17f7b52a020aa509d26f42b/apps/aetx/src/aetx.erl#L160-L166
https://github.com/aeternity/epoch/blob/17a22c1e02dfae53f17f7b52a020aa509d26f42b/apps/aecontract/src/aect_create_tx.erl
https://github.com/aeternity/protocol/blob/437c2faffd16b372e54edcc6967ee6fc2181bef0/serializations.md#table-of-object-tags